ROADMAP.md

```diff
@@ -1,6 +1,26 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.8 Release, Planned for release: Oct 2023
+## Kubeflow 1.9 Release, Planned for release: Jul 2024
+The Kubeflow Community plans to deliver its v1.9 release in Jul 2024 per this [timeline](https://github.com/kubeflow/community/blob/master/releases/release-1.9/README.md#timeline). The high level deliverables are tracked in the [v1.9 Release](https://github.com/orgs/kubeflow/projects/61) Github project board. The v1.9 release process will be managed by the v1.9 [release team](https://github.com/kubeflow/community/blob/master/releases/release-1.9/release-team.md) using the best practices in the [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md).
+
+### Themes
+* Kubernetes 1.29 support
+* CNCF Graduation
+* LLM APIs
+* New component: Model Registry
+* Kubeflow Pipelines and kfp-tekton merged in a single GitHub repository
```

Comment:
This is not what we have planned and it should be discussed. Let's remove this for now.

Reply:
cc @chensun @terrytangyuan

Reply:
cc @rimolive @Tomcli

Reply:
According to our last discussion, I'll resolve this one as we made an agreement of the ktp-tekton merge

Reply:
Maybe add V2 to the release statement because we are still maintaining kfp-tekton v1 in a separate repo. Just to let others know kfp-tekton is still in active maintenance mode.